pyhole/core/plugin.py

```diff
@@ -178,6 +178,7 @@ def load_plugins(*args, **kwargs):
             __import__("pyhole.plugins", globals(), locals(), [plugin_name])
         except Exception, exc:
             LOG.error(exc)
+            raise
```

Comment:
There was a reason I didn't re-raise here, but of course I failed to leave a comment, so I have no idea what it was.

Did you come across an issue that caught your attention?